### DISCUSSION TITLE
OSError: [WinError -2147023728]

### DISCUSSION THREAD

**Discussion by dreamdomroy** [pywinrt/python-winsdk#18] [Q&A] [answered]
There's an app that tells me who's calling through a Windows's system notification, I want to concatenate it with my python program.

here is the code I found on the internet:

```python
from winsdk.windows.ui.notifications.management import *
from winsdk.windows.ui.notifications import *

listener = UserNotificationListener.get_current()

def handler(listener, event):
    notification = listener.get_notification(event.user_notification_id)
    if hasattr(notification, "app_info"):
        print("App Name: ", notification.app_info.display_info.display_name)

listener.add_notification_changed(handler)   
```

I test it on both Windows 11 and Windows 10, it keeps showing this error message : **OSError: [WinError -2147023728]**
 any idea?

**Comment by dlech** [comment, marked as answer]
Please see #16. The `windows.ui` namespace it not usable outside of a packaged WinUI 3 app.
